src/base/bittorrent/torrenthandle.cpp

```diff
@@ -1055,6 +1081,23 @@ qreal TorrentHandle::maxRatio(bool *usesGlobalRatio) const
     return ratioLimit;
 }
 
+int TorrentHandle::maxSeedingTime(bool *usesGlobalSeedingTime) const
```

Comment:
I would use `std::pair<int, bool> maxSeedingTime()` and then the function would be:
```C++
std::pair<int, bool> TorrentHandle::maxSeedingTime() const
{
    return m_seedingTimeLimit == USE_GLOBAL_SEEDING_TIME ?
        std::make_pair(m_session->globalMaxSeedingMinutes(), true) :
        std::make_pair(m_seedingTimeLimit, false) :
}